doc/nrf/releases_and_maturity/software_maturity.rst

```diff
@@ -202,6 +202,8 @@ The following table indicates the software maturity levels of the support for ea
      - --
      - --
```

Comment:
These entries could also be removed. This will also fix the doc build error. This is with the understanding that Bluetooth has support for nRF54L05 and L10.

Reply:
now fixed